src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -454,7 +454,9 @@ def _data_sourcing_request_sender(self) -> Sender[ComponentMetricRequest]:
                 name="Data Pipeline: Data Sourcing Actor Request Channel"
             )
             actor = DataSourcingActor(
-                request_receiver=channel.new_receiver(limit=_REQUEST_RECV_BUFFER_SIZE),
+                request_receiver=channel.new_receiver(
+                    limit=_REQUEST_RECV_BUFFER_SIZE, name=channel.name + " Receiver"
+                ),
```

Comment:
Wait! This shouldn't be necessary. Maybe you are using `name` directly? Is you use `str(receiver)` or `repr(receiver)` the channel name (and the word "receiver") will be already used:

* https://github.com/frequenz-floss/frequenz-channels-python/blob/b3c98e594310edde687bde4d9b39f4ac370ed676/src/frequenz/channels/_broadcast.py#L472-L483
* https://github.com/frequenz-floss/frequenz-channels-python/blob/b3c98e594310edde687bde4d9b39f4ac370ed676/src/frequenz/channels/_broadcast.py#L291-L303

Reply:
I was using .name directly, yes

Reply:
Removed